RELEASE.md

```diff
@@ -0,0 +1,41 @@
+# Release
+Follow the steps outlined in this document to release a new version of `cloudgov-style`.
+
+### Blah blah
+- Releases should follow [semantic versioning](http://semver.org/).
+- The Github tags for the release should just be the number, no `v` before.
+- The project will be pushed to [npm](https://www.npmjs.com/package/cloudgov-style) and [RubyGems](https://rubygems.org/gems/cloudgov-style).
```

Comment:
I'm confused about this, there isn't any sort of automatic publishing of packages using a tag or something like that, correct? By what mechanism does one "push" the project to these two registries?